examples/slack-clone/nextjs-slack-clone-dotenvx/README.md

```diff
@@ -0,0 +1,192 @@
+# Integrating Dotenvx with a Supabase Project
+
+This project is a full-stack Slack clone built using:
+
+### Frontend:
+
+- **[Next.js](https://github.com/vercel/next.js):** A React framework optimized for production.
+- **[Supabase.js](https://supabase.com/docs/library/getting-started):** For user management and real-time data syncing.
+
+### Backend:
+
+- **[Supabase](https://supabase.com/dashboard):** A hosted Postgres database with a RESTful API, used alongside Supabase.js.
+- **GitHub Authentication:** For user login.
+
+---
+
+## Introduction
+
+This example demonstrates how to use [dotenvx](https://dotenvx.com/) and `config.toml` to manage multiple environments seamlessly. You'll learn how to set up local and production environments with shared, secure configurations.
+
+---
+
+## Core Concept
+
+`config.toml` supports environment variables through the `env()` syntax. Using `dotenvx`, you can securely manage sensitive values like GitHub credentials for third-party authentication.
+
+### Key Features of Dotenvx:
+
+- Secrets are encrypted and stored securely, while private decryption keys are saved in `.env.keys` (excluded from version control).
+- Teams can share private keys to decrypt environment values securely.
+- Learn more: [Dotenvx secrets and encryption](https://dotenvx.com/encryption).
+
+This example guides you through deploying and managing app environments with dotenvx.
+
+---
+
+## Structuring Environment Files
+
+Follow the conventions used in this project, where environments are split into three files:
+
+1. `supabase/.env.development` - For local development using `npx supabase start`.
```

Comment:
```suggestion
1. `supabase/.env.local` - For local development using `npx supabase start`.
```

By convention, the .development suffix is reserved for another layer of override, like `.env.staging.development`.

Reply:
Sorry I was confused. It's the other way round.

Reply:
Hum, right that's what I had originally. But the cli doesn't look-up for `.env.local` by default. So I tough using `.env.development` might be a better alternative.